containers/codespaces-linux/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -70,6 +70,9 @@ apt-get-update-if-needed()
 
 # Run install apt-utils to avoid debconf warning then verify presence of other common developer tools and dependencies
 if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
+    # Needed for adding manpages-posix and manpages-posix-dev which are non-free packages in Debian
+    CODENAME=$(lsb_release -cs)
+    echo -e "deb http://deb.debian.org/debian/ $(CODENAME) main contrib non-free \ndeb http://deb.debian.org/debian/ $(CODENAME)-updates main contrib non-free \ndeb http://deb.debian.org/debian-security/ $(CODENAME)/updates main contrib non-free \ndeb http://deb.debian.org/debian/ $(CODENAME)-backports main contrib non-free" >/etc/apt/sources.list
```

Comment:
@samruddhikhandale This can will actually break Ubuntu, so the addition of the Debian non-free sources should first verify that the distro is actually Debian before doing this.  (You can use `if [ "$(lsb_release -is)" = "Debian" ]; then`). Ubuntu does not appear to put `manpages-posix` in a separate package feed, so it works by default and you can skip this.

 You can test this using the "ubuntu" and "debian" definitions.
 
 You should probably also **append** to `/etc/apt/sources.list` rather than replace just to be safe (use `>>` not `>`).  This will probably remove the existing "free" sources which is not what you want.

Reply:
Update: The Debian only comment still applies.. but I realized the append is not right since this is just adding to the existing values.

Reply:
Added the condition and tested with both definitions, works fine

![image](https://user-images.githubusercontent.com/24955913/110497711-53055f00-80bc-11eb-971d-943dfb10f3b2.png)
![image](https://user-images.githubusercontent.com/24955913/110497763-5e588a80-80bc-11eb-868b-267133448d77.png)